docs/security.md

```diff
@@ -12,7 +12,7 @@ This document outlines our security policy for the codebases, platforms that we
 We appreciate responsible disclosure of vulnerabilities that might impact the integrity of our platforms and users. In the interest of saving everyone time, we encourage you to report vulnerabilities with these in mind:
 
 1. Ensure that you are using the **latest**, **stable**, and **updated** versions of the Operating System and Web Browser(s) available to you on your machine.
-2. We consider using tools & online utilities to report issues with SPF & DKIM configs, SSL Server tests, etc., in the category of ["beg bounties"](https://www.troyhunt.com/beg-bounties) and are unable to respond to these reports.
+2. We consider using tools & online utilities to report issues with SPF & DKIM configs, SSL Server tests, etc., in the category of ["bUg bounties"](https://www.troyhunt.com/beg-bounties) and are unable to respond to these reports.
```

Comment:
`U` should be lowercase, no?

Reply:
Also, somewhat confusingly, this actually isn't a typo -- look at the linked article.